sources/csv/data.go

```diff
@@ -40,12 +44,33 @@ type Column struct {
 type Table struct {
 	Table_name    string   `json:"table_name"`
 	File_patterns []string `json:"file_patterns"`
-	Columns       []Column `json:"columns"`
 }
 
-// LoadManifest reads the manifest file and unmarshalls it into a list of Table struct.
+func GetCSVFiles(conv *internal.Conv, sourceProfile profiles.SourceProfile) (tables []Table, err error) {
+	// If manifest file not provided, we assume the csvs exist in the same directory
+	// in table_name.csv format.
+	if sourceProfile.Csv.Manifest == "" {
+		for t := range conv.SpSchema {
```

Comment:
add a logger at all decision points like this, regarding which approach was chosen